### ISSUE TITLE
Colab Notebook not generating transforms.json file for custom data

### ISSUE THREAD

**Issue by niconielsen32** [nerfstudio-project/nerfstudio#2272]
Have tried running the colab notebook for some days now without success. The notebook have been working for some weeks but suddenly stopped. When I try to upload and process the data it only creates the image folders but not the transforms.json file. It stops running after the image extraction phase.

Have been trying some different wrap arounds and using instant-ngp to generate the file and then manually upload it and reformat the transform file. Doesn't seem to work either. A bit weird since most versions for the dependencies are locked.

Any possible solutions?

**Comment by Tianchong-Jiang** [comment]
I had the same issue. Have you tried to comment out all the lines with "CONSOLE" in the data processing part in nerfstudio? It seems to me that whenever there is an error and a call of CONSOLE, the program exits without error trace backs. In this case, there is a call of CONSOLE to print "Done copying images".

**Comment by yankeesong** [comment]
I'm having the same issue. I dig a little bit and found that COLMAP installation has errors. I don't want to dig further since I'm not an expert. Could someone fix this please?

**Comment by yixiang1120** [comment]
Same issue here. Could someone fix this please? Thanks!

**Comment by sudoku-lord** [comment]
Yep, I'm getting the same issue in a notebook which previously had no issues with generating transforms.json. Was there some kind of update in the code which affected COLMAP versioning? @tancik (sorry for the tag, just hoping for a quick resolution)

**Comment by tancik** [comment]
It sounds like out provided compiled colmap is no longer installing correctly. Likely some underlying colab dependencies changed. In the meantime you can try installing colmap using these instructions - https://gist.github.com/kwea123/f0e8f38ff2aa94495dbfe7ae9219f75c
